mnelab.py

```diff
@@ -388,10 +388,13 @@ def set_montage(self):
             self.all.current.montage = name
             self.all.data[self.all.index].montage = name
             montage = mne.channels.read_montage(name)
-            self.all.current.raw.set_montage(montage)
-            self.all.data[self.all.index].raw.set_montage(montage)
-            self._update_infowidget()
-            self._toggle_actions()
+            try: 
```

Comment:
Here a check if there is at least one channel name in the data and montage would probably be better than a try/except block. Then a dialog should pop up with an error message.